torch/csrc/dynamo/guards.h

```diff
@@ -8,3 +9,67 @@ PyObject* torch_c_dynamo_guards_init();
 // not visible there.
 void* convert_to_root_guard_manager(py::object root);
 bool run_root_guard_manager(void* root, PyObject* f_locals);
+
+struct LocalState {
```

Comment:
Should we add namespace to avoid naming collision?

Reply:
Yes, we should, and the namespace should be `namespace torch::dynamo`. I'm just not sure about the namespace scope. Should we add `convert_to_root_guard_manager` and `run_root_guard_manager` to the namespace as well? @anijain2305

Reply:
Let's move everything to `torch::dynamo` scope

Reply:
Thanks, I will refine it.